packages/devtools_app/lib/src/ui/utils.dart

```diff
@@ -178,6 +178,24 @@ class _OffsetScrollbarState extends State<OffsetScrollbar> {
   }
 }
 
+void scrollToPosition(
+  ScrollController scrollController,
+  double position,
+) {
+  //TODO I would add padding here so if a row is half out of screen it will still jump
+  final scrollStart = scrollController.offset;
+  final scrollEnd = scrollStart + scrollController.position.extentInside;
```

Comment:
nit: maybe rename to visibleStart and visibleEnd to indicate that these values are not the min / max scroll extents, but rather the visible range.

Bonus - use the Range class in DevTools so that you can use Range.contains to check if the position is inside the visibleRange